tests/paramsServer.test.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable max-len */
+'use strict';
+
+const { Server } = require('http');
+const { createServer } = require('../src/createServer.js');
+const axios = require('axios');
+
+const PORT = 5701;
+const HOST = `http://localhost:${PORT}`;
+
+describe('Params server', () => {
+  describe('createServer', () => {
+    describe('basic scenarios', () => {
+      it('should create a server', () => {
+        expect(createServer)
+          .toBeInstanceOf(Function);
+      });
+
+      it('should create an instance of Server', async() => {
+        expect(createServer())
+          .toBeInstanceOf(Server);
+      });
+    });
+
+    describe('Server', () => {
+      let server;
+
+      beforeAll(() => {
+        server = createServer();
+        server.listen(PORT);
+      });
+
+      afterAll(() => {
+        server.close();
+      });
```

Comment:
Why do you do it after all but not each?
What if 1 test breaks the server 
The others will also fail

Reply:
agree, fixed